website/docs/r/cognitive_account_customer_managed_key.html.markdown

```diff
@@ -0,0 +1,118 @@
+  custom_subdomain_name = "example-account"
+  identity {
+    type         = "SystemAssigned, UserAssigned"
+    identity_ids = [azurerm_user_assigned_identity.example.id]
+  }
+}
+resource "azurerm_key_vault" "example" {
+  name                     = "example-vault"
+  location                 = azurerm_resource_group.example.location
+  resource_group_name      = azurerm_resource_group.example.name
+  tenant_id                = data.azurerm_client_config.current.tenant_id
+  sku_name                 = "standard"
+  soft_delete_enabled      = true
+  purge_protection_enabled = true
+}
+resource "azurerm_key_vault_access_policy" "cognitive" {
+  key_vault_id       = azurerm_key_vault.example.id
+  tenant_id          = azurerm_cognitive_account.example.identity.0.tenant_id
+  object_id          = azurerm_cognitive_account.example.identity.0.principal_id
+  key_permissions    = ["get", "create", "list", "restore", "recover", "unwrapkey", "wrapkey", "purge", "encrypt", "decrypt", "sign", "verify"]
+  secret_permissions = ["get"]
+}
+resource "azurerm_key_vault_access_policy" "client" {
+  key_vault_id       = azurerm_key_vault.test.id
+  tenant_id          = data.azurerm_client_config.current.tenant_id
+  object_id          = data.azurerm_client_config.current.object_id
+  key_permissions    = ["get", "create", "delete", "list", "restore", "recover", "unwrapkey", "wrapkey", "purge", "encrypt", "decrypt", "sign", "verify"]
+  secret_permissions = ["get"]
+}
+resource "azurerm_key_vault_access_policy" "user" {
+  key_vault_id       = azurerm_key_vault.test.id
+  tenant_id          = azurerm_user_assigned_identity.example.tenant_id
+  object_id          = azurerm_user_assigned_identity.example.principal_id
+  key_permissions    = ["get", "create", "delete", "list", "restore", "recover", "unwrapkey", "wrapkey", "purge", "encrypt", "decrypt", "sign", "verify"]
+  secret_permissions = ["get"]
+}
+resource "azurerm_key_vault_key" "example" {
+  name         = "example-key"
+  key_vault_id = azurerm_key_vault.test.id
+  key_type     = "RSA"
+  key_size     = 2048
+  key_opts     = ["decrypt", "encrypt", "sign", "unwrapKey", "verify", "wrapKey"]
+  depends_on = [
+    azurerm_key_vault_access_policy.cognitive,
+    azurerm_key_vault_access_policy.client,
+    azurerm_key_vault_access_policy.user,
+  ]
+}
+resource "azurerm_cognitive_account_customer_managed_key" "example" {
+  cognitive_account_id = azurerm_cognitive_account.example.id
+  key_vault_key_id     = azurerm_key_vault_key.example.id
+  identity_client_id   = azurerm_user_assigned_identity.example.client_id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `cognitive_account_id` - (Required) The ID of the Cognitive Account. Changing this forces a new resource to be created.
+
+* `key_vault_key_id` - (Required) The ID of the Key Vault Key.
+
+* `identity_client_id` - (Optional) The client id of the user assigned identity that has access to the key.
```

Comment:
```suggestion
* `identity_client_id` - (Optional) The Client ID of the User Assigned Identity that has access to the key.
```

presumably this only needs to be specified when there's multiple identities attached to the Cognitive Account? If so, can we add a note to that effect